contributor/03-contributor-ethos.md

```diff
@@ -0,0 +1,64 @@
+# Draft comments
+
+ Outline for this section
+
+     Contributors by definition work in teams that they are not familiar with. This section explains how to avoid common pitfalls when crossing team boundaries.
+
+ ## Article
+
+In the last segment we have motivated why you would want to reuse components and become active as a Contributor. This artice is sharing best practices on how to successfully contribute your changes to the host team's code base.
+
+ ### Contributor Ethos
+
+An InnerSource Contributor trying to make a contribution to the host team essentially is a guest in their home. In general a good guest is expected to behave in a certain way:
+
+* They knock at the door.
+* They anticipate and follow house rules.
+* They understand they are not the home owner and acty accordingly.
+
+How do these expectations apply to InnerSource projects?
+
+#### Getting in
+
+In the real world their are several ways that guests are invited in:
+
+* An "open" sign lets them know they can walk right in without knocking.
+* Knock on the door unexpected.
+* Knock on the door only when invited.
+
+With InnerSource projects there are several ways that code gets in:
+
+Sometimes you just open a pull request. At times you may need to get a feature request accepted before opening the pull request that fulfills it.
```

Comment:
> At times you may need to get a feature request accepted before opening the pull request that fulfills it.

This point is made with more detail in the paragraph `In the same way that ...`.  Suggest just deleting this sentence here since the point will be brought up later.

Reply:
If we remove this sentence, then the case that sometimes a simple, well written PR could solve things would go missing. Maybe just rephrasing this would help